targets/targets.json

```diff
@@ -2786,6 +2786,21 @@
         ],
         "device_name": "STM32L072CZ"
     },
+    "MCU_STM32L071xx": {
+        "inherits": [
+            "MCU_STM32L0"
+        ],
+        "public": false,
+        "extra_labels_add": [
+            "STM32L0",
+            "STML071xx"
+        ],
+        "device_has_remove": [
+            "ANALOGOUT",
+            "TRNG"
+        ],
+        "device_name": "STM32L071xx"
```

Comment:
In fact, you have to remove "device_name" from "MCU_STM32L071xx",
as device name is the exact used MCU, and STM32L071xx is the family

This "device_name" has to be defined at target level, so in the custom_target.json file.
This parameter is used for bootloader info, and for debug with pyOCD and MBED Studio for ex.
It has to be part of https://github.com/ARMmbed/mbed-os/blob/master/tools/arm_pack_manager/index.json
So maybe STM32L071CZTx for you ?

Reply:
ok, got it